source/includes/fundamentals/code-snippets/CRUD/projection.go

```diff
@@ -98,9 +109,13 @@ func main() {
 
 	fmt.Println("\nAggregation Projection:\n")
 	{
+		// Creates a projection stage to include only the "title" and
+		// "course_id" fields
 		// begin aggregate projection
 		projectStage := bson.D{{"$project", bson.D{{"title", 1}, {"course_id", 1}}}}
 
+		// Aggregates results and prints them as structs with the
+		// specified projection
```

Comment:
S: for consistency, maybe change this to "...prints them as structs including only the specified fields"